interfaces/connector_lock.yaml

```diff
@@ -0,0 +1,13 @@
+description: >-
+  This interface defines one connector locking motor (e.g. for AC sockets with no fixed attached cable)
+cmds:
+  lock:
+    description: Lock connector lock
+  unlock:
+    description: unlock connector lock (e.g. normal unlock or enforced by OCPP)
+vars:
+  error:
+    description: Error events for locking motor
```

Comment:
what about unlocking - is this purposely left out?

Reply:
No, the error events is a merge of the MRC codes and some additional ones found in the ChargeByte manual. Which ones would make sense to add here? The error handling will be refactored before this is merged once the new error handling is ready. But we are already collecting useful error types in this enum

Reply:
this var has been removed in the refactoring